src/main/java/practice/StreamPractice.java

```diff
@@ -50,7 +76,11 @@ public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toA
      */
     public List<Person> getWorkablePeople(int fromAge, int femaleToAge,
                                           int maleToAge, List<Person> peopleList) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(person -> person.getAge() >= fromAge
+                        && ((person.getSex() == Person.Sex.MAN && person.getAge() <= maleToAge)
+                        || (person.getSex() == Person.Sex.WOMAN && person.getAge() <= femaleToAge)))
+                .collect(Collectors.toList());
```

Comment:
Can be simplified to:

```suggestion
        return peopleList.stream()
                .filter(p -> p.getAge() >= fromAge
                        && (p.getSex() == Person.Sex.MAN
                        ? p.getAge() <= maleToAge
                        : p.getAge() <= femaleToAge))
                .toList();
```